bodhi/server/consumers/updates.py

```diff
@@ -194,9 +194,8 @@ def work_on_bugs(self, session, update, bugs):
                 log.info("Commenting on %r" % bug.bug_id)
                 comment = config['initial_bug_msg'] % (
```

Comment:
if we choose to define the comment in models.py `modified` method this is not needed anymore.